### DISCUSSION TITLE
Alias/Relative paths not working for webpack 5. I am still using the import-glob plugin.

### DISCUSSION THREAD

**Discussion by knightswatch3** [webpack/webpack#12954] [Q&A]
Would the **`import-glob`** plugin still work with webpackv5 ? I receive the file-references errors stating the relative paths are not being able to resolve.

Not sure why am I receiving the errors for file references:

`resolve: {
    extensions: [".js", ".jsx", ".json"],
    alias: {
      Assets: path.resolve(__dirname, "..", "..", "src/client/assets"),
      Config: path.resolve(__dirname, "..")
    },
    modules: [path.resolve(__dirname, "..", "..", "src/client/client_modules"), "node_modules"],
    preferAbsolute: false
  }`

This is my resolve key:value

**Comment by alexander-akait** [comment]
`import-glob plugin` we don't have this plugin, can't help you without more information

**Reply by knightswatch3** [reply]
Apologies. This is a module(loader) which was loading all my scss files and unified them in one single file called main.scss. So what I am trying to understand is, what changes in webpack 5 if I use the same module.

https://www.npmjs.com/package/import-glob

Would it stop working ?

**Reply by alexander-akait** [reply]
Should work, what is the error?

**Reply by knightswatch3** [reply]
I don't receive any errors but the imports in the main scss file seem not to happen. Which why my scss file throughout the project complain about failed to extend the css classes from libraries imported by main.scss file

**Reply by alexander-akait** [reply]
Check which imports `import-glob` generate, debug it inside loader
